### PR TITLE
Nulana.Remotix version 6.4.4.11749

### PR DESCRIPTION
**Pull request by JasonCook599**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41093)

### CODE REVIEW

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Nulana.Remotix
+PackageVersion: 6.4.4.11749
```
Comment (ItzLevvie) [inline manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.installer.yaml:5]:
```suggestion
PackageVersion: 6.4.4
```

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Nulana.Remotix
+PackageVersion: 6.4.4.11749
```
Comment (ItzLevvie) [inline manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.yaml:5]:
```suggestion
PackageVersion: 6.4.4
```

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Nulana.Remotix
+PackageVersion: 6.4.4.11749
```
Comment (ItzLevvie) [inline manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.4.4
```

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.installer.yaml:5]: This must match Control Panel.

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.locale.en-US.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.locale.en-US.yaml:5]: This must match Control Panel.

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.yaml:5]: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+  InstallLocation: /Dir=<INSTALLPATH>
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://downloads.remotix.com/rx-win/Remotix-6.4.4-11749-installer.exe
+  InstallerSha256: E22F80FF8E81575290C7C193933DC412676CA7CAAAE29585FE2B6E8EE2542999
```
Comment (ItzLevvie) [inline manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.installer.yaml:14]:
```suggestion
  InstallerSha256: E22F80FF8E81575290C7C193933DC412676CA7CAAAE29585FE2B6E8EE2542999
  Scope: user
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+PublisherSupportUrl: https://remotix.com/help/
+PrivacyUrl: https://remotix.com/pp/
+Author: Nulana
+PackageName: Remotix
+PackageUrl: https://remotix.com/remotix-win/
+License: Copyright (c) 2011-2021 Nulana. All rights reserved.
+LicenseUrl: https://remotix.com/tos/
+Copyright: Copyright (c) 2011-2021 Nulana. All rights reserved.
+CopyrightUrl: https://remotix.com/tos/
+ShortDescription: Remotix
```
Comment (ItzLevvie) [inline manifests/n/Nulana/Remotix/6.4.4.11749/Nulana.Remotix.locale.en-US.yaml:18]:
```suggestion
ShortDescription: Connect to all your computers from anywhere.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the version folder as well.

You can press `.` (dot) on your keyboard on this PR to bring up Visual Studio Code:
image
image

**Comment by JasonCook599** [comment]
> You'll also need to update the version folder as well.

The folder has been updated.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**